composer.json

```diff
@@ -28,26 +28,26 @@
     ],
     "license": "GPL-2.0-or-later",
     "require": {
-        "php": "^7.2",
+        "php": ">7.2",
```

Comment:
Should we use "^7.2 || ~8.0" ?

Reply:
I changed my mind on PHP version constraints and now think that having an upper boundary is actually more harmful than it helps, at least for a "not that much legacy" codebase.

Therefore I will go with using a minimum constraint only like [Symfony](https://github.com/symfony/symfony/blob/b5d4b33d5e65d956194c170fda5f8f40bc6a921a/composer.json#L35)

Reply:
Yeah, it's a weighing process. "If PHP 8 series does not provide any breaking changes" we should be safe :D